server/bin/pbench-verify-backup-tarballs.py

```diff
@@ -290,38 +290,41 @@ def main():
 
     logger = get_pbench_logger(_NAME_, config)
 
-    archive = Path(config.ARCHIVE).resolve()
-    if not archive:
+    archive = Path(config.ARCHIVE)
+    try:
+        archivepath = archive.resolve(strict=True)
+    except FileNotFoundError:
         logger.error(
             "The ARCHIVE directory {}, does not resolve to a real location", archive
         )
         return 1
-
-    if not archive.is_dir():
-        logger.error(
-            "The setting for ARCHIVE in the config file is {}, but that is"
-            " not a directory",
-            archive,
-        )
-        return 1
+    else:
+        if not archivepath.is_dir():
+            logger.error(
+                "The setting for ARCHIVE in the config file is {}, but that is"
+                " not a directory",
+                archive,
+            )
+            return 1
 
     # add a BACKUP field to the config object
     config.BACKUP = backup = config.conf.get("pbench-server", "pbench-backup-dir")
-    backuppath = Path(backup).resolve()
-    if not backuppath:
+    try:
+        backuppath = Path(backup).resolve(strict=True)
+    except FileNotFoundError:
```

Comment:
This exception will need a different errror logged (""The backup directory {}, does not resolve to a real location").

Reply:
+1